css/properties/color.json

```diff
@@ -0,0 +1,539 @@
+{
+  "css": {
+    "properties": {
+      "color": {
```

Comment:
We need a basic support block.

Reply:
The original table doesn't have a basic support row. What should constitute basic support, in this case? A version that supports one or more of the color values? Supporting all of the CSS level 1 values? Or just assume they're all `true`?

Reply:
All true (1 and 4 for Firefox, Edge: 12, Chrome: 1). Color was in the original css implementations (wasn't version 1 for IE or Opera, but for Chrome, Firefox, and Edge yes.

Reply:
Thank you for these details! I've added the basic support block.